worker/entry.sh

```diff
@@ -1,2 +1,25 @@
 #!/bin/sh
+
+if lsmod | grep '\<tun\>' > /dev/null; then
+	tun_minor=$(grep tun /proc/misc | cut -d ' ' -f1)
+	mkdir -p /dev/net
+	if ! mknod -m 666 /dev/net/tun c 10 "${tun_minor}"; then
+		echo "Unable to create TUN device node"
+		exit 1
+	fi
+else
+	echo "TUN module not loaded, unable to setup networking"
+	exit 1
```

Comment:
I want to approve this for the kvm stuff but now I'm missing tun networking, so I'm glad you added this check.
```
worker_1  | TUN module not loaded, unable to setup networking
leviathan-generic-test_worker_1 exited with code 1
```
When I get a chance I'll try to enable the tun module on an ec2 spot instance and hopefully approve this PR.

Reply:
@klutchell I've received this same error running the worker after a fresh boot, try loading the module before running the worker with `modprobe tun`.

Reply:
It seems that `lsmod | grep '\<tun\>'` will fail if the module is builtin.
```
modprobe tun
```
```
lsmod | grep tun
```
```
modinfo tun 
name:           tun
filename:       (builtin)
alias:          devname:net/tun
alias:          char-major-10-200
license:        GPL
file:           drivers/net/tun
author:         (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
description:    Universal TUN/TAP device driver
```